vmm.mk

```diff
@@ -6,7 +6,7 @@
 # Snippet to build libvmm.a, to be included in a full-system Makefile.
 # Needs the variable LionsOS to point to the top of the LionsOS tree.
 #
-V3_BOARDS := BOARD_imx8mm_evk
+V3_BOARDS := imx8mm_evk maaxboard
```

Comment:
Since you're changing this at the same time, can you also make this `GIC_V3_BOARDS` instead?

Reply:
Done